gnome/gnome-calculator/Portfile

```diff
@@ -42,8 +45,8 @@ depends_run         port:adwaita-icon-theme
 # blacklist compilers that do not support C11 (redefinition of typedef ‘GtkSourceTag’ at gtksourceview/gtksourcetag.h:35)
 compiler.blacklist  *gcc-3.* *gcc-4.* {clang < 300}
 
-configure.args      --disable-schemas-compile \
-                    --disable-silent-rules
+configure.cflags-append     -Wno-incompatible-pointer-types -Wno-incompatible-function-pointer-types
+configure.cxxflags-append   -Wno-incompatible-pointer-types -Wno-incompatible-function-pointer-types
```

Comment:
These imply that there's a bug in the code itself because there shouldn't be assignments between incompatible pointer types. Have you looked at whether the errors are legitimate?

Reply:
The old version also gives this error on modern compilers.  There's already an upstream issue report: https://gitlab.gnome.org/GNOME/gnome-calculator/-/issues/380

Reply:
Sure, but if it's only a couple of places, a patch should be fast to create. Just look at what is failing; the error message is usually pretty explicit.

Reply:
I wouldn't be able to quicklyake a patch for 2 reasons:

1. Looking at the linked issue report, the number of these errors is extremely high, so a patch would likely be a decently complex undertaking.
2. I've never worked with Vala before, so that adds to the time required.

I'm sure a patch will come soon enough from upstream as more Linux distros move to GCC 14, but IMHO it should not be a blocker for this update (especially since the current version doesn't compile due to the same issue).